### PR TITLE
Improve note formatting

### LINKED ISSUE #47: Notes should support newlines
## Input

```

Previous commitments:
- Mike: log food (yes)
- Jacob: Meditate in the morning (No)
- Daniel: Meditate (yes)
- Eric: Evening routine (yes)
```

## Output

image

### PR DESCRIPTION
**Pull request by americanhanko**
## This PR

- Resolves #47
- Notes now support basic HTML (newlines, style tags, etc.)
- Add a new `note.scss` stylesheet to make note author and body a bit more attractive
- Use the `simple_format` Rails helper method with a `class` keyword argument to specify the style sheet class
- Alphabetize imports in `application.scss`

## Example

Screen Shot 2020-11-08 at 10 53 33 PM

### CODE REVIEW

File: app/views/calls/_note.html.erb
Comment (thestrabusiness) [inline app/views/calls/_note.html.erb:2]: We don't need `simple_format` for the author's name because we're not trying to transform a string into HTML. The method wraps the given string in `<p>` tags, so I think this can be:
```suggestion
  <p class="note-author"><%= note.author.first_name %></p>
```

File: app/assets/stylesheets/note.scss
Comment (thestrabusiness) [inline app/assets/stylesheets/note.scss:1]: This is ok for now, but it would be nice to make a pass at consolidating our colors and sizes into a variables files eventually.

**Review by thestrabusiness — DISMISSED** [review]
Looks good. I like the new styling.

**Comment by thestrabusiness** [comment]
Once you rebase CI should pass.

**Review by thestrabusiness — DISMISSED** [review]

File: spec/system/user_creates_note_spec.rb
Comment (thestrabusiness) [reply spec/system/user_creates_note_spec.rb:16]: We don't even need `#to_s`, this can just be `have_content(user.first_name)`

**Comment by americanhanko** [comment]
Need approval from @thestrabusiness or @kurtisnotcurtis before we can merge.

**Review by thestrabusiness — APPROVED** [review]

**Pull request merged**
